docs/application/vscode-ext/web.md

```diff
@@ -333,46 +333,106 @@ To debug application, follow these steps:
+You must select the **HTML extension file** in the file explorer before you launch Design Editor.
 
-- Command Palette
-    1. In VS Code, click **View** > **Command Palette**.
-    2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+### Run Design Editor
+
+To run Design Editor, perform one of the following:
 
-        A new tab is created and the Design Editor is launched.
+- Using Command Palette
+    1. To launch Design Editor, in VS Code, click **View** > **Command Palette**.
+    2. In the input field of **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
 
-- Design-Editor button
+        A new tab is created and Design Editor is launched.
 
-    To launch the Design Editor, click **Design-Editor** at the bottom right of the screen.
+- Using Design-Editor button
+
+    To launch Design Editor, click **Design-Editor** on the VS Code window as shown in the following figure:
 
     ![Design Editor button](media/tizen_web27.png)
-- Right-click preference
-    1. Select the HTML file that you want to launch and right-click.
+
+- Using right-click options
+    1. To launch Design Editor, select the HTML file that you want to launch and right-click.
     2. From the list that appears, click **Tizen Web: Run Design-Editor**.
 
         ![Design Editor right click](media/tizen_web28.png)
 
-Design Editor is successfully launched.
+### Features
 
-![Design Editor launch](media/tizen_web29.png)
+Following are the WYSIWYG Design Editor features:
 
-### Quick Action Execution
-Right-click on the **project folder** to use the extension function in the menu.
+- Drag and Drop a component
 
-For more information on each function, see:
-- [Tizen Web: Build Tizen App](#Build-Project)
-- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
-- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
-- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
-- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+    You can drag and drop a component from the Widget palette to the canvas.
+
+    ![Design Editor Widget](media/tizen_web35.png)
+    
+- Change component properties
+
+    After placing the UI components, you can set the component properties in the attribute palette.
+
+    ![Design Editor Attribute](media/tizen_web36.png)
+
+- Preview
+
+    You can use **Preview** ![Design Editor PreviewIcon](media/tizen_web45.png) in the toolbar to display the preview of the edited page.
+
+    ![Design Editor Preview](media/tizen_web37.png)
+   
+- JavaScript Assistant
+
+   You can use **JavaScript Assistant** ![Design Editor JSAssitantIcon](media/tizen_web46.png) in the toolbar to edit the Javascript code with **JavaScript Assistant Wizard**.
+
+    ![Design Editor JavaScript Assistant](media/tizen_web38.png)
```

Comment:
@singa2000 Question. In this image, the next icon to "JavaScript Assistant" is also marked. 
Would you please let me know the reason?

Reply:
For launch the "Javascript Assistant Wizard" it need to select the  </> icon.

Reply:
Thanks for the image update @singa2000.

However, I can see the image icons are different.

![image](https://user-images.githubusercontent.com/9958374/74002134-6f166880-49b1-11ea-90b6-ab8789dfcae9.png)


If possible, how about unifying the size of icon images?
I think the size of extensions.png is good. @Reni-90 Would you please let me know your opinion?

Reply:
@mijin85cho 
I completely agree with your suggestion. It is good to maintain one resolution and size for all images. The size of the image, extensions.png is good.